index.html

```diff
@@ -2081,7 +2071,7 @@ <h3>Securing Mechanisms</h3>
 
         <p>
 An <dfn class="export">embedded proof</dfn> is a mechanism where the proof is
-included in the serialization of the data model. One such RECOMMENDED embedded
+included in the serialization of the data model. One such embedded
```

Comment:
Why removing this?

The preceding paragraph on enveloping proof uses the same terminology to 'recommend' jose-cose. This paragraph does the same for DI. Either *both* _RECOMMENDED_ statements are removed, or none of them. Personally, I prefer keeping it as is: after all, we are 'recommending' the two proof mechanisms that this WG standardizes, which is perfectly fine, and it certainly does not exclude third parties coming up with their own.

Reply:
Agreed. Do not remove this, the WG is explicitly recommending at least the securing mechanisms that it is publishing as standards.

Reply:
both paragraphs should be consistent